creator-node/docker-compose.yml

```diff
@@ -30,8 +30,26 @@ services:
     networks:
       - creator-node-network
 
+  filebeat:
+    extends:
+      file: ../common-services.yml
+      service: filebeat
+    env_file:
+      - override.env
+    networks:
+      - creator-node-network
+
+  metricbeat:
+    extends:
+      file: ../common-services.yml
+      service: metricbeat
+    env_file:
+      - override.env
+    networks:
+      - creator-node-network
+
   backend:
-    container_name: server
+    container_name: creator-node
```

Comment:
Same comments as below, we should not be doing multiple tasks in the same PR

Reply:
Without these changes the purpose of this PR, to enable logging for ELK, will not be functional.